41_snapshots-btrfs

```diff
@@ -311,24 +374,24 @@ detect_microcode()
 path_snapshot()
 {
 	case "${path_snapshot}" in
-		true) name_snapshot=("${snap_dir_name}");;
-		*) name_snapshot=("${snap_dir_name#*"/"}")
+		true) name_snapshot=("${snap_full_name}");;
+		*) name_snapshot=("${snap_full_name#*"/"}")
```

Comment:
I'm not sure whether this still works, as I'm not sure what is the expected outcome for this case. 

For me even on `master` the snapshots are called `/snapshots/13/snapshot`, and this line converted such name into `snapshots/13/snapshot`.

If this doesn't work as you want it to work, let me know and give me an example of a test case 🙂

Reply:
hi,
it's been a while since I looked at the code.
this part was supposed to display only the name given to the snapshot instead of the full path
(valid only in grub menu)

for example : 
I don't use snapper.
I put all my snapshots in a subvolume named "@snapshots" at the root of the filesystem.
and I name my snapshots like this: @03-01-2018-15:08_test1 (@date_+name)

Resulting in : 
* full path = @snapshots/@03-01-2018-15:08_test1
* only name = @03-01-2018-15:08_test1

in grub: 
* full path
>submenu 'Snapshot: 2018-01-03 15:08:41 @snapshots/@03-01-2018-15:08_test1
* only name
>submenu 'Snapshot: 2018-01-03 15:08:41 @03-01-2018-15:08_test1'

I understand that in your case, it doesn't work as expected.

Reply:
Got it, so it's because the subvolume name starts with `@` in your case (`@snapshots`) and with `/` in my case (`/.snapshots`) - then I think it should still work as expected.

Please test how this code works on your side, that would be the best proof that I didn't accidentally break anything - and if everything is OK, let's merge 🎉

Reply:
everything is OK.
thank you for your skills
It's a nice gift you're giving this year.